onnxruntime/server/core/metric_registry.h

```diff
@@ -0,0 +1,113 @@
+#pragma once
+
+#include <prometheus/registry.h>
+
+namespace onnxruntime {
+namespace server {
+struct MetricRegistry {
+  // returns the global metric registry instance
+  // use this handle to get the instance to increment/count requests
+  static MetricRegistry& Get() {
```

Comment:
Would you mind make this struct/class (why not use class here?) a regular class? When using it, use a `unique_ptr` of it.